.gitignore

```diff
@@ -1,4 +1,4 @@
-### Python template
+ ### Python template
```

Comment:
Do not make changes in the `.gitignore` file, your PR (pull request) should contain only `app/main.py` file